src/test/java/com/esri/arcgisruntime/toolkit/skins/FeatureTemplatePickerTilePaneSkinUnitTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.skins;
+
+import com.esri.arcgisruntime.toolkit.FeatureTemplatePicker;
+import javafx.application.Platform;
+import javafx.geometry.Orientation;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Feature template picker tile pane skin unit tests.
+ */
+@DisplayName("feature template picker tile pane skin unit tests")
+public class FeatureTemplatePickerTilePaneSkinUnitTest {
+
+  @BeforeAll
+  static void startPlatform() {
+    try {
+      Platform.startup(() -> {});
+    } catch (Exception ex) {
+      // toolkit already initialized
```

Comment:
@colinanderson it shouldn't fail though. I do this because when multiple tests classes are run, the first one will need to run Platform.startup but the others don't. I can't think of a better way to do that.